packages/cli/src/commands/setup/jobs/templates/jobs.ts.template

```diff
@@ -0,0 +1,32 @@
+// Setup for background jobs. Jobs themselves live in api/src/jobs
+// Execute jobs in dev with `yarn rw jobs work`
+// See https://docs.redwoodjs.com/docs/background-jobs
+
+import { PrismaAdapter, JobManager } from '@redwoodjs/jobs'
+
+import { db } from 'src/lib/db'
+import { logger } from 'src/lib/logger'
+
+export const jobs = new JobManager({
+  adapters: {
+    prisma: new PrismaAdapter({ db, logger }),
+  },
+  queues: ['default'] as const,
+  logger,
+  workers: [
+    {
+      adapter: 'prisma',
+      logger,
+      queue: '*',
```

Comment:
Let's add a comment here to say what '*' means if you think that's sensible? Something like:
```
queue: '*', // Listens on queues
```